pkg/node/local.go

```diff
@@ -776,58 +776,43 @@ func DestroyCurrentIfLocalNetwork(app *application.Avalanche) error {
 		}
 	}
 	if currentlyRunningRootDir == "" {
-		return nil
+		return false, "", nil
 	}
 	localClusters, err := listLocalClusters(app, nil)
 	if err != nil {
-		return fmt.Errorf("failed to list local clusters: %w", err)
+		return false, "", fmt.Errorf("failed to list local clusters: %w", err)
 	}
 	for clusterName, rootDir := range localClusters {
 		clusterConf, err := app.GetClusterConfig(clusterName)
 		if err != nil {
-			return fmt.Errorf("failed to get cluster config: %w", err)
+			return false, "", fmt.Errorf("failed to get cluster config: %w", err)
 		}
 		network := models.ConvertClusterToNetwork(clusterConf.Network)
 		if rootDir == currentlyRunningRootDir && network.Kind == models.Local {
-			_ = DestroyLocalNode(app, clusterName)
+			return true, clusterName, nil
 		}
 	}
-	return nil
+	return false, "", nil
 }
 
-func StopCurrentIfLocalNetwork(app *application.Avalanche) error {
-	ctx, cancel := utils.GetANRContext()
-	defer cancel()
-	currentlyRunningRootDir := ""
-	cli, _ := binutils.NewGRPCClientWithEndpoint( // ignore error as ANR might be not running
-		binutils.LocalClusterGRPCServerEndpoint,
-		binutils.WithAvoidRPCVersionCheck(true),
-		binutils.WithDialTimeout(constants.FastGRPCDialTimeout),
-	)
-	if cli != nil {
-		status, _ := cli.Status(ctx) // ignore error as ANR might be not running
-		if status != nil && status.ClusterInfo != nil {
-			if status.ClusterInfo.RootDataDir != "" {
-				currentlyRunningRootDir = status.ClusterInfo.RootDataDir
-			}
-		}
+func DestroyCurrentIfLocalNetwork(app *application.Avalanche) error {
+	isLocal, clusterName, err := CurrentIsLocalNetwork(app)
+	if err != nil {
+		return err
 	}
-	if currentlyRunningRootDir == "" {
-		return nil
+	if isLocal {
+		_ = DestroyLocalNode(app, clusterName)
 	}
-	localClusters, err := listLocalClusters(app, nil)
+	return nil
+}
+
+func StopCurrentIfLocalNetwork(app *application.Avalanche) error {
```

Comment:
nit: better name can be StopLocalNetwork which returns error if it's not running or not local 

pls feel free to ignore this comment